Source/Analytics/Web/src/components/healthRisk/HealthRiskSelector.js

```diff
@@ -48,19 +47,22 @@ export default class HealthRiskSelector extends Component {
 
     render() {
         return (
-            <div className="tableContainer">
-                <h2 className="headline">Reports for
-                <Select className="headline-select"
+            <>
+                <h2 className="headerText">Reports for
+                <Select className="headerSelect"
                         value={this.state.selected}
                         onChange={this.saveSelectedValue.bind(this)}
                     >
                         {this.renderOptions()}
                     </Select>
                 </h2>
-
-                <h5>Reports the last 7 days</h5>
-                <ReportsPerHealthRiskPerDay healthRisk={this.state.selected} />
-            </div>
+                <div className="tableContainer">
+                    <h5>Reports for the last 7 days</h5>
+                    <ReportsPerHealthRiskPerDay healthRisk={this.state.selected} />
```

Comment:
Should this be `selectedHealthRisk`? It would make a lot of sense to make this state, and the `ReportsPerHealthRiskPerDay` props match in naming.